src/adapter/controller/ArticleController.ts

```diff
@@ -54,10 +59,33 @@ export class ArticleController {
       res.status(HTTP_STATUS_CODE.OK).send(JSON.stringify(response))
     } catch (error) {
       if (error instanceof NotFoundError) {
-        res.status(HTTP_STATUS_CODE.NotFound).send(error.message)
+        res.status(HTTP_STATUS_CODE.NotFound).send(JSON.stringify({ message: error.message }))
       } else if (error instanceof DataBaseError) {
-        res.status(HTTP_STATUS_CODE.DataBaseError).send(error.message)
-      } else res.status(HTTP_STATUS_CODE.InternalServerError).send(`${error}`)
+        res.status(HTTP_STATUS_CODE.DataBaseError).send(JSON.stringify({ message: error.message }))
+      } else
+        res
+          .status(HTTP_STATUS_CODE.InternalServerError)
+          .send(JSON.stringify({ message: `${error}` }))
+    }
+  }
+
+  public async editArticle(res: Response, req: Request) {
+    try {
+      const { body } = new EditArticleRequest(req)
+      const article = await this.articleService.edit(req.params.id, body)
+      const response = new ArticleIdResponse(article)
+      res.status(HTTP_STATUS_CODE.OK).send(JSON.stringify(response))
+    } catch (error) {
+      if (error instanceof NotFoundError) {
+        res.status(HTTP_STATUS_CODE.NotFound).send(JSON.stringify({ message: error.message }))
+      } else if (error instanceof DataBaseError) {
+        res.status(HTTP_STATUS_CODE.DataBaseError).send(JSON.stringify({ message: error.message }))
+      } else if (error instanceof QueryBodyError) {
+        res.status(HTTP_STATUS_CODE.QueryBodyError).send(JSON.stringify({ message: error.message }))
+      } else
+        res
+          .status(HTTP_STATUS_CODE.InternalServerError)
+          .send(JSON.stringify({ message: `${error}` }))
```

Comment:
見逃していましたができれば else の後の処理は `{}` で囲ってほしいです。読みやすさの意味で
```suggestion
      } else {
        res
          .status(HTTP_STATUS_CODE.InternalServerError)
          .send(JSON.stringify({ message: `${error}` }))
      }
```